nginx-1.21.4-wolfssl.patch

```diff
@@ -65,10 +63,22 @@ index 80be906e..8767aa33 100644
    --with-openssl-opt=OPTIONS         set additional build options for OpenSSL
  
 diff --git a/src/event/ngx_event_openssl.c b/src/event/ngx_event_openssl.c
-index 84afecd0..fe7e328e 100644
---- a/src/event/ngx_event_openssl.c
-+++ b/src/event/ngx_event_openssl.c
-@@ -353,6 +353,8 @@ ngx_ssl_create(ngx_ssl_t *ssl, ngx_uint_t protocols, void *data)
+--- a/src/event/ngx_event_openssl.c	2021-12-22 12:04:40.978789957 -0500
++++ b/src/event/ngx_event_openssl.c	2021-12-22 12:13:23.031389526 -0500
+@@ -20,10 +20,12 @@
+ 
+ static X509 *ngx_ssl_load_certificate(ngx_pool_t *pool, char **err,
+     ngx_str_t *cert, STACK_OF(X509) **chain);
++#if !defined(WOLFSSL_NGINX) || !defined(HAVE_LIBOQS)
+ static EVP_PKEY *ngx_ssl_load_certificate_key(ngx_pool_t *pool, char **err,
```

Comment:
What is the issue with `ngx_ssl_load_certificate_key` and `ngx_ssl_password_callback`? Oh you are calling `SSL_CTX_use_PrivateKey_file` directly... Might make comment here in code.

Reply:
Sure.  I can do that.